### ISSUE TITLE
Improve the TCB Table

### ISSUE THREAD

**Issue by fitzthum** [confidential-containers/confidentialcontainers.org#56]
The docs now have a threat model page that has a table showing how each platform measures the 4 components of the CoCo TCB. You can find it [here](https://confidentialcontainers.org/docs/architecture/trust-model/trust-model/) in the `measurement details` section. I wasn't really sure about how TDX and SE work for this. Can experts on these platforms please check what I wrote in this table and make sure it is true? @BbolroC @mythi

**Comment by BbolroC** [comment]
Yes, I will get back to you if I find anything that needs to be changed. Thanks for the documentation. ❤️
